tests/copy-paste.js

```diff
@@ -630,6 +630,53 @@ require([
                    "double_trouble should not be valid");
         });
 
+        it("should not update broken references on paste into group", function () {
+            util.loadXML("");
+            paste([
+                ["id", "type", "labelItext:en-default"],
+                ["/group", "Group", "group"],
+            ]);
+            paste([
+                ["id", "type", "calculateAttr"],
+                ["/four", "DataBindOnly", "2 + /data/two"],
+            ]);
+            var four = util.getMug("group/four");
+            assert.equal(four.p.calculateAttr, "2 + /data/two");
+            assert(!util.isTreeNodeValid(four), "four should not be valid");
+        });
+
+        it("should update question references on paste into group", function () {
```

Comment:
Is your plan for these two tests to treat the root as `/data/`, so `/group` will be transformed to `/data/group`? and resolve that way? That seems like it could get pretty messy.

Seems like something we would want to test out with FMs though. Not sure what they would expect to happen, although I think not changing anything in calculate or similar fields is the easiest/safest way to go.